modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/RocksDbStorageEngine.java

```diff
@@ -125,40 +137,49 @@ public String name() {
 
     @Override
     public void start() throws StorageException {
-        registerDataRegion(DEFAULT_DATA_REGION_NAME);
+        registerDataRegion(engineConfig.defaultRegion().value());
 
         // TODO: IGNITE-17066 Add handling deleting/updating data regions configuration
         engineConfig.regions().listenElements(new ConfigurationNamedListListener<>() {
             @Override
             public CompletableFuture<?> onCreate(ConfigurationNotificationEvent<RocksDbDataRegionView> ctx) {
-                registerDataRegion(ctx.newName(RocksDbDataRegionView.class));
+                RocksDbDataRegionView newValue = ctx.newValue();
+
+                assert newValue != null;
```

Comment:
I think this has already been tested in the configuration module and checking is not needed.

Reply:
This is mostly because IDEA's nullability warning is triggered, I prefer to suppress it using an assertion

Reply:
ok